test/battle/ability/adaptability.c

```diff
@@ -0,0 +1,46 @@
+#include "global.h"
+#include "test/battle.h"
+
+TO_DO_BATTLE_TEST("Adaptability increases same-type attack bonus from x1.5 to x2");
+
+SINGLE_BATTLE_TEST("(TERA) Terastallizing into a different type with Adaptability gives 2.0x STAB", s16 damage)
```

Comment:
Is there a reason to move those 2 tests? This to me looks to more like a Tera test then Adaptability.